src/main/java/com/atlassian/db/replica/api/state/State.java

```diff
@@ -14,6 +14,7 @@ private State(String name) {
     public static final State MAIN = new State("MAIN");
     public static final State REPLICA = new State("REPLICA");
     public static final State CLOSED = new State("CLOSED");
+    public static final State COMMITED_MAIN = new State("COMMITED_MAIN");
```

Comment:
BTW why this is class, not "enum"?

Reply:
[Enums are hard to evolve](https://github.com/wyrzyk/java-api-design-checklist/blob/a63b78efe8a988343ecdb7bfaf3f81079f5ebae2/CHECKLIST.md#255-do-not-add-new-values-to-a-released-enumeration)

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html
> You should use enum types any time you need to represent a fixed set of constants. That includes natural enum types such as the planets in our solar system and data sets where you know all possible values at compile time—for example, the choices on a menu, command line flags, and so on.

We don't know the fixed set of states yet. Using enum would block [the API's evolution](https://github.com/wyrzyk/java-api-design-checklist/blob/master/CHECKLIST.md#255-do-not-add-new-values-to-a-released-enumeration).